OpenShift/helm/nexus-iq/templates/deployment.yaml

```diff
@@ -23,9 +23,9 @@ spec:
         image: {{ .Values.iq.imageName }}:{{ .Values.iq.imageTag }}
         ports:
         - containerPort: {{ .Values.iq.applicationPort }}
-        {{- if .Values.deployment.terminationGracePeriodSeconds }}
-        terminationGracePeriodSeconds: {{ .Values.deployment.terminationGracePeriodSeconds }}
-        {{- end }}
+        # {{- if .Values.deployment.terminationGracePeriodSeconds }}
+        # terminationGracePeriodSeconds: {{ .Values.deployment.terminationGracePeriodSeconds }}
+        # {{- end }}
```

Comment:
please delete unused code rather than commenting it out (unless it's in 'values.yml' where it can serve as a sample for how to configure things)